base_comment_template/README.rst

```diff
@@ -14,23 +14,33 @@ Base Comments Templates
     :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
     :alt: License: AGPL-3
 .. |badge3| image:: https://img.shields.io/badge/github-OCA%2Freporting--engine-lightgray.png?logo=github
-    :target: https://github.com/OCA/reporting-engine/tree/14.0/base_comment_template
+    :target: https://github.com/OCA/reporting-engine/tree/13.0/base_comment_template
```

Comment:
It looks like you generated the readme for branch 13.0

Reply:
Although bot will regenerated it, i have changed.

Reply:
This has not been changed at the end

Reply:
Please regenerate this for not having this double diff